minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -53,22 +69,25 @@ public static void main(String[] args) throws Exception {
     CommonUtils.sleepMs(Constants.SECOND_MS);
   }
 
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private BlockWorker mWorker = null;
   private long mWorkerCapacityBytes;
   private int mUserBlockSize;
   private int mQuotaUnitBytes;
   private String mTachyonHome;
-  private String mWorkerDataFolder;
   private Thread mWorkerThread = null;
   private String mLocalhostName = null;
   private LocalTachyonMaster mMaster;
   private TachyonConf mMasterConf;
```

Comment:
It's possible to have all conf instance pointing to the same object. I still keep them separately for two reasons: (1) technically, those conf should be in different JVM and independent from each other; (2) we may want to add tests in the future where the conf diverge. Storing them separately enables these kinds of test.